### PR TITLE
[13.0][ADD] stock_inventory_line_open

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
This module adds the capability to navigate to inventory lines on validated inventory adjustments.

@ForgeFlow

### CODE REVIEW

**Review by HviorForgeFlow — APPROVED** [review]
Functional and code review
LGTM
:+1:

File: stock_inventory_line_open/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from odoo import fields, models
+
+
+class StockInventory(models.Model):
+    _name = "stock.inventory"
+    _inherit = ["stock.inventory", "mail.thread"]
+
+    state = fields.Selection(tracking=True)
+
+    def _track_subtype(self, init_values):
```
Comment (rousseldenis) [inline stock_inventory_line_open/models/stock.py]: Maybe you could add something in README for that no ?

Comment (LoisRForgeFlow) [reply stock_inventory_line_open/models/stock.py]: ups, that is dead code (not imported), I will remove it.

**Review by rousseldenis — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
One more detail, I corrected the corpyright year and set the development status to beta.

**Pull request merged**